pandas/core/indexes/datetimes.py

```diff
@@ -1904,14 +1904,17 @@ def delete(self, loc):
 
     def tz_convert(self, tz):
         """
-        Convert tz-aware DatetimeIndex from one time zone to another (using
-        pytz/dateutil)
+        Convert tz-aware DatetimeIndex from one time zone to another.
+
+        When using DatetimeIndex providing with timezone this method
```

Comment:
This is a cumbersome sentence and is pretty duplicative of the above, I would remove it.

Reply:
I regret the auto typing mistake, I wanted to convey that i will surely be doing it as per recommended by you.
Please pardon my Indian English.

Reply:
@hammadmashkoor no problem. thanks for the PR!

Reply:
I don't think this extended summary is needed at all as its duplicated of the Summary.